diff -urN aaa/gnome-system-tools-0.24.0/src/common/xst-platform.c gnome-system-tools-0.24.0/src/common/xst-platform.c
--- aaa/gnome-system-tools-0.24.0/src/common/xst-platform.c	Fri Nov 22 19:24:55 2002
+++ gnome-system-tools-0.24.0/src/common/xst-platform.c	Sun Apr 27 21:39:39 2003
@@ -37,6 +37,7 @@
 extern GdkPixbuf *suse;
 extern GdkPixbuf *freebsd;
 extern GdkPixbuf *gentoo;
+extern GdkPixbuf *pld;
 
 XstPlatform *
 xst_platform_new (const gchar *key, const gchar *name)
@@ -148,6 +149,8 @@
 		return freebsd;
 	else if (g_ascii_strncasecmp (platform->name, "Gentoo", 6) == 0)
 		return gentoo;
+	else if (g_ascii_strncasecmp (platform->name, "PLD", 3) == 0)
+		return pld;
 	else return NULL;
 }
 
diff -urN aaa/gnome-system-tools-0.24.0/src/common/xst-tool.c gnome-system-tools-0.24.0/src/common/xst-tool.c
--- aaa/gnome-system-tools-0.24.0/src/common/xst-tool.c	Wed Feb 19 19:31:19 2003
+++ gnome-system-tools-0.24.0/src/common/xst-tool.c	Sun Apr 27 21:43:06 2003
@@ -64,6 +64,7 @@
 GdkPixbuf *suse;
 GdkPixbuf *freebsd;
 GdkPixbuf *gentoo;
+GdkPixbuf *pld;
 
 enum {
 	BOGUS,
@@ -553,6 +554,7 @@
 	suse = gdk_pixbuf_new_from_file (PIXMAPS_DIR "/suse.png", NULL);
 	freebsd = gdk_pixbuf_new_from_file (PIXMAPS_DIR "/freebsd.png", NULL);
 	gentoo = gdk_pixbuf_new_from_file (PIXMAPS_DIR "/gentoo.png", NULL);
+	pld = gdk_pixbuf_new_from_file (PIXMAPS_DIR "/pld.png", NULL);
 }
 
 static void
