diff -urN gnome-system-tools-0.91.0.orig/backends/configure.in gnome-system-tools-0.91.0/backends/configure.in
--- gnome-system-tools-0.91.0.orig/backends/configure.in	2004-08-20 15:07:43.766322128 +0200
+++ gnome-system-tools-0.91.0/backends/configure.in	2004-08-20 15:08:01.346649512 +0200
@@ -33,7 +33,7 @@
 GETTEXT_PACKAGE=setup-tools-backends
 AC_SUBST(GETTEXT_PACKAGE)
 
-ALL_LINGUAS="az ar bn bs ca cs da de el en_CA en_GB es eu fi gu hr ja ko lt ms nb nl no pa pl pt pt_BR ro sq sr sr@Latn sv ta uk zh_CN zh_TW"
+ALL_LINGUAS="az ar bn bs ca cs da de el en_CA en_GB es eu fi gu hr ja ko lt ms nb nl pa pl pt pt_BR ro sq sr sr@Latn sv ta uk zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 AC_OUTPUT([
diff -urN gnome-system-tools-0.91.0.orig/configure.in gnome-system-tools-0.91.0/configure.in
--- gnome-system-tools-0.91.0.orig/configure.in	2004-08-20 15:07:43.658338544 +0200
+++ gnome-system-tools-0.91.0/configure.in	2004-08-20 15:08:22.745396408 +0200
@@ -161,7 +161,7 @@
 dnl Transtlations
 dnl ========================================================
 
-ALL_LINGUAS="am ar az bg bn ca cs da de el en_CA en_GB es et eu fi fr ga gl gu hr hu it ja ko lt lv mk ml mn ms nl nn no pa pl pt pt_BR ro ru sk sq sr sr@Latn sv ta tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am ar az bg bn ca cs da de el en_CA en_GB es et eu fi fr ga gl gu hr hu it ja ko lt lv mk ml mn ms nb nl nn pa pl pt pt_BR ro ru sk sq sr sr@Latn sv ta tr uk vi wa zh_CN zh_TW"
 GETTEXT_PACKAGE=gnome-system-tools
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [gettext package name])
