diff -urN aa/gnome-system-tools-0.91.0/backends/configure.in gnome-system-tools-0.91.0/backends/configure.in
--- aa/gnome-system-tools-0.91.0/backends/configure.in	2004-08-20 12:39:59.000000000 +0200
+++ gnome-system-tools-0.91.0/backends/configure.in	2004-08-22 20:01:57.196692656 +0200
@@ -3,9 +3,8 @@
 dnl Process this file with autoconf to produce a configure script.
 dnl ==============================================================
 AC_INIT(system-tools-backends,1.0.0, http://bugzilla.gnome.org/enter_bug.cgi?product=GNOME%20System%20Tools)
-AM_INIT_AUTOMAKE(AC_PACKAGE_NAME, AC_PACKAGE_VERSION)
-
 AC_CONFIG_AUX_DIR(.)
+AM_INIT_AUTOMAKE(AC_PACKAGE_NAME, AC_PACKAGE_VERSION)
 
 AC_PROG_INTLTOOL([0.29])
 
