diff -urN aaa/gnome-system-tools-0.30.0/src/boot/boot.desktop.in.in gnome-system-tools-0.30.0/src/boot/boot.desktop.in.in
--- aaa/gnome-system-tools-0.30.0/src/boot/boot.desktop.in.in	2003-09-25 17:39:56.000000000 +0200
+++ gnome-system-tools-0.30.0/src/boot/boot.desktop.in.in	2003-12-12 17:05:26.000000000 +0100
@@ -3,7 +3,8 @@
 _Comment=Configure the system boot process
 Exec=boot-admin
 Icon=
-Terminal=0
+Terminal=false
 Type=Application
 Categories=Application;System;
 StartupNotify=true
+Encoding=UTF-8
diff -urN aaa/gnome-system-tools-0.30.0/src/network/network.desktop.in.in gnome-system-tools-0.30.0/src/network/network.desktop.in.in
--- aaa/gnome-system-tools-0.30.0/src/network/network.desktop.in.in	2003-09-25 17:39:56.000000000 +0200
+++ gnome-system-tools-0.30.0/src/network/network.desktop.in.in	2003-12-12 17:06:22.000000000 +0100
@@ -3,7 +3,8 @@
 _Comment=Configure network devices and connections
 Exec=network-admin
 Icon=
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;System
+Categories=Application;System;
 StartupNotify=true
+Encoding=UTF-8
diff -urN aaa/gnome-system-tools-0.30.0/src/services/services.desktop.in.in gnome-system-tools-0.30.0/src/services/services.desktop.in.in
--- aaa/gnome-system-tools-0.30.0/src/services/services.desktop.in.in	2003-09-25 17:39:57.000000000 +0200
+++ gnome-system-tools-0.30.0/src/services/services.desktop.in.in	2003-12-12 17:06:58.000000000 +0100
@@ -3,7 +3,7 @@
 _Comment=Configure which services will be run when the system starts
 Exec=services-admin
 Icon=
-Terminal=0
+Terminal=false
 Type=Application
 Categories=Application;System;
-
+Encoding=UTF-8
diff -urN aaa/gnome-system-tools-0.30.0/src/time/time.desktop.in.in gnome-system-tools-0.30.0/src/time/time.desktop.in.in
--- aaa/gnome-system-tools-0.30.0/src/time/time.desktop.in.in	2003-09-25 17:39:58.000000000 +0200
+++ gnome-system-tools-0.30.0/src/time/time.desktop.in.in	2003-12-12 17:07:27.000000000 +0100
@@ -3,7 +3,8 @@
 _Comment=Change system time, date, and timezone
 Exec=time-admin
 Icon=
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;System
+Categories=Application;System;
 StartupNotify=true
+Encoding=UTF-8
diff -urN aaa/gnome-system-tools-0.30.0/src/users/users.desktop.in.in gnome-system-tools-0.30.0/src/users/users.desktop.in.in
--- aaa/gnome-system-tools-0.30.0/src/users/users.desktop.in.in	2003-09-25 17:40:00.000000000 +0200
+++ gnome-system-tools-0.30.0/src/users/users.desktop.in.in	2003-12-12 17:08:01.000000000 +0100
@@ -3,7 +3,8 @@
 _Comment=Add or remove users and groups
 Exec=users-admin
 Icon=
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;System
+Categories=Application;System;
 StartupNotify=true
+Encoding=UTF-8
