diff -ur gnome-system-tools-1.2.0.orig/src/boot/boot.desktop.in.in gnome-system-tools-1.2.0/src/boot/boot.desktop.in.in
--- gnome-system-tools-1.2.0.orig/src/boot/boot.desktop.in.in	2005-01-10 01:12:26.000000000 +0100
+++ gnome-system-tools-1.2.0/src/boot/boot.desktop.in.in	2005-03-30 14:13:40.000000000 +0200
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/boot.png
 Terminal=false
 Type=Application
-Categories=Application;System;Settings
+Categories=GTK;GNOME;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=boot-admin
diff -ur gnome-system-tools-1.2.0.orig/src/disks/disks.desktop.in.in gnome-system-tools-1.2.0/src/disks/disks.desktop.in.in
--- gnome-system-tools-1.2.0.orig/src/disks/disks.desktop.in.in	2005-01-10 01:12:26.000000000 +0100
+++ gnome-system-tools-1.2.0/src/disks/disks.desktop.in.in	2005-03-30 14:13:40.000000000 +0200
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/disks.png
 Terminal=false
 Type=Application
-Categories=Application;System;Settings
+Categories=GTK;GNOME;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=disks-admin
diff -ur gnome-system-tools-1.2.0.orig/src/network/network.desktop.in.in gnome-system-tools-1.2.0/src/network/network.desktop.in.in
--- gnome-system-tools-1.2.0.orig/src/network/network.desktop.in.in	2005-01-10 01:12:26.000000000 +0100
+++ gnome-system-tools-1.2.0/src/network/network.desktop.in.in	2005-03-30 14:13:40.000000000 +0200
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/network.png
 Terminal=false
 Type=Application
-Categories=Application;System;Settings
+Categories=GTK;GNOME;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=network-admin
diff -ur gnome-system-tools-1.2.0.orig/src/services/services.desktop.in.in gnome-system-tools-1.2.0/src/services/services.desktop.in.in
--- gnome-system-tools-1.2.0.orig/src/services/services.desktop.in.in	2005-01-10 01:12:27.000000000 +0100
+++ gnome-system-tools-1.2.0/src/services/services.desktop.in.in	2005-03-30 14:13:40.000000000 +0200
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/services.png
 Terminal=false
 Type=Application
-Categories=Application;System;Settings
+Categories=GTK;GNOME;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=runlevel-admin
diff -ur gnome-system-tools-1.2.0.orig/src/shares/shares.desktop.in.in gnome-system-tools-1.2.0/src/shares/shares.desktop.in.in
--- gnome-system-tools-1.2.0.orig/src/shares/shares.desktop.in.in	2005-01-10 01:12:27.000000000 +0100
+++ gnome-system-tools-1.2.0/src/shares/shares.desktop.in.in	2005-03-30 14:15:00.000000000 +0200
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/shares.png
 Terminal=false
 Type=Application
-Categories=Application;System;Settings
+Categories=GTK;GNOME;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=shares-admin
diff -ur gnome-system-tools-1.2.0.orig/src/time/time.desktop.in.in gnome-system-tools-1.2.0/src/time/time.desktop.in.in
--- gnome-system-tools-1.2.0.orig/src/time/time.desktop.in.in	2005-01-10 01:12:27.000000000 +0100
+++ gnome-system-tools-1.2.0/src/time/time.desktop.in.in	2005-03-30 14:13:40.000000000 +0200
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/time.png
 Terminal=false
 Type=Application
-Categories=Application;System;Settings
+Categories=GTK;GNOME;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=time-admin
diff -ur gnome-system-tools-1.2.0.orig/src/users/users.desktop.in.in gnome-system-tools-1.2.0/src/users/users.desktop.in.in
--- gnome-system-tools-1.2.0.orig/src/users/users.desktop.in.in	2005-01-10 01:12:27.000000000 +0100
+++ gnome-system-tools-1.2.0/src/users/users.desktop.in.in	2005-03-30 14:13:40.000000000 +0200
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/users.png
 Terminal=false
 Type=Application
-Categories=Application;System;Settings
+Categories=GTK;GNOME;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=users-admin
