diff -urN aa/gnome-system-tools-0.33.0/src/common/CommonMakefile gnome-system-tools-0.33.0/src/common/CommonMakefile
--- aa/gnome-system-tools-0.33.0/src/common/CommonMakefile	1970-01-01 01:00:00.000000000 +0100
+++ gnome-system-tools-0.33.0/src/common/CommonMakefile	2004-05-18 12:35:44.085205040 +0200
@@ -0,0 +1,16 @@
+desktop = $(toolname).desktop
+
+$(desktop).in: %.desktop.in: %.desktop.in.in
+	sed s#Icon=.*#Icon=$(pixmapsdir)/$(toolname).png# < $< > $(toolname).desktop.in
+
+$(desktop): %.desktop: %.desktop.in
+		../../intltool-merge -d ../../po $(desktop).in $(desktop)
+
+desktopdir = $(datadir)/applications/
+desktop = $(toolname).desktop
+desktop_DATA = $(desktop)
+
+startheredir = $(sysconfdir)/X11/sysconfig/
+desktop = $(toolname).desktop
+starthere_DATA = $(desktop)
+
