diff -urN aaa/gnome-system-tools-0.29.0/backends/time-conf.in gnome-system-tools-0.29.0/backends/time-conf.in
--- aaa/gnome-system-tools-0.29.0/backends/time-conf.in	2003-10-22 20:57:08.000000000 +0200
+++ gnome-system-tools-0.29.0/backends/time-conf.in	2003-10-30 13:56:35.000000000 +0100
@@ -666,7 +666,8 @@
          table =>
              [
               [ "timezone",     \&time_set_rh62_zone, [LOCAL_TIME, ZONEINFO] ],
-              [ "local_time",   \&gst_replace_join_all, NTP_CONF, "server", "[ \t]+" ],
+              [ "local_time",   \&time_set_local_time ],
+              [ "sync",         \&gst_replace_join_all, NTP_CONF, "server", "[ \t]+" ],
               [ "sync_active",  \&gst_service_sysv_set_status, 90, "ntpd" ],
               ]
                 }
