diff -urN gnome-system-tools-2.13.2.orig/src/boot/boot.desktop.in.in gnome-system-tools-2.13.2/src/boot/boot.desktop.in.in
--- gnome-system-tools-2.13.2.orig/src/boot/boot.desktop.in.in	2006-02-18 23:34:43.803832000 +0100
+++ gnome-system-tools-2.13.2/src/boot/boot.desktop.in.in	2006-02-19 00:02:29.293832000 +0100
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/boot.png
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;System;Settings;
+Categories=GNOME;GTK;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=boot-admin
diff -urN gnome-system-tools-2.13.2.orig/src/disks/disks.desktop.in.in gnome-system-tools-2.13.2/src/disks/disks.desktop.in.in
--- gnome-system-tools-2.13.2.orig/src/disks/disks.desktop.in.in	2006-02-18 23:34:43.803832000 +0100
+++ gnome-system-tools-2.13.2/src/disks/disks.desktop.in.in	2006-02-19 00:02:05.303832000 +0100
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/disks.png
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;System;Settings;
+Categories=GNOME;GTK;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=disks-admin
diff -urN gnome-system-tools-2.13.2.orig/src/network/network.desktop.in.in gnome-system-tools-2.13.2/src/network/network.desktop.in.in
--- gnome-system-tools-2.13.2.orig/src/network/network.desktop.in.in	2006-02-18 23:34:43.803832000 +0100
+++ gnome-system-tools-2.13.2/src/network/network.desktop.in.in	2006-02-19 00:01:43.583832000 +0100
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/network.png
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;System;Settings;
+Categories=GNOME;GTK;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=network-admin
diff -urN gnome-system-tools-2.13.2.orig/src/services/services.desktop.in.in gnome-system-tools-2.13.2/src/services/services.desktop.in.in
--- gnome-system-tools-2.13.2.orig/src/services/services.desktop.in.in	2006-02-18 23:34:43.813832000 +0100
+++ gnome-system-tools-2.13.2/src/services/services.desktop.in.in	2006-02-19 00:02:54.893832000 +0100
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/services.png
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;System;Settings;
+Categories=GNOME;GTK;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=runlevel-admin
diff -urN gnome-system-tools-2.13.2.orig/src/shares/shares.desktop.in.in gnome-system-tools-2.13.2/src/shares/shares.desktop.in.in
--- gnome-system-tools-2.13.2.orig/src/shares/shares.desktop.in.in	2006-02-18 23:34:43.813832000 +0100
+++ gnome-system-tools-2.13.2/src/shares/shares.desktop.in.in	2006-02-19 00:02:47.813832000 +0100
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/shares.png
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;System;Settings;
+Categories=GNOME;GTK;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=shares-admin
diff -urN gnome-system-tools-2.13.2.orig/src/time/time.desktop.in.in gnome-system-tools-2.13.2/src/time/time.desktop.in.in
--- gnome-system-tools-2.13.2.orig/src/time/time.desktop.in.in	2006-02-18 23:34:43.813832000 +0100
+++ gnome-system-tools-2.13.2/src/time/time.desktop.in.in	2006-02-18 23:59:20.093832000 +0100
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/time.png
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;System;Settings;
+Categories=GNOME;GTK;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=time-admin
diff -urN gnome-system-tools-2.13.2.orig/src/users/users.desktop.in.in gnome-system-tools-2.13.2/src/users/users.desktop.in.in
--- gnome-system-tools-2.13.2.orig/src/users/users.desktop.in.in	2006-02-18 23:34:43.813832000 +0100
+++ gnome-system-tools-2.13.2/src/users/users.desktop.in.in	2006-02-18 23:58:55.753832000 +0100
@@ -5,7 +5,7 @@
 Icon=@pixmapsdir@/users.png
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;System;Settings;
+Categories=GNOME;GTK;System;Settings;X-Capplet;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-system-tools
 X-GNOME-Bugzilla-Component=users-admin
